src/CashuWallet.ts

```diff
@@ -410,47 +484,65 @@ class CashuWallet {
 	/**
 	 * Creates blinded messages for a given amount
 	 * @param amount amount to create blinded messages for
+	 * @param preference optional preference for splitting proofs into specific amounts. overrides amount param
+	 * @param count? optionally set count to derive secret deterministically. CashuWallet class must be initialized with seed phrase to take effect
 	 * @returns blinded messages, secrets, rs, and amounts
 	 */
 	private createRandomBlindedMessages(
 		amount: number,
-		amountPreference?: Array<AmountPreference>
+		amountPreference?: Array<AmountPreference>,
+		count?: number
+	): BlindedMessageData & { amounts: Array<number> } {
+		const amounts = splitAmount(amount, amountPreference);
+		return this.createBlindedMessages(amounts, count);
+	}
+
+	/**
+	 * Creates blinded messages for a according to @param amounts
```

Comment:
review message description, `... for a according ...`

Reply:
let's look at this again after we change the function